### ISSUE TITLE
How to compute a jax-compatible time of year?

### ISSUE THREAD

**Issue by meteorologytoday** [google/jax-datetime#14]
I need to compute the time of year using jax_datetime as a float in the interval [0, 1), which is computed as follows:

1. `total_days_of_year(dt: jax_datetime.Datetime) -> jax.typing.ArrayLike`: Returns the total days of the year specified by `dt`.
2. `day_of_year(dt: jax_datetime.Datetime) -> jax.typing.ArrayLike`: Returns the day of the year specified by `dt`.
3. The desired `normalized_day_of_year(dt: jax_datetime.Datetime) = day_of_year(dt) / total_days_of_year(dt)`.

 I need this resulting float to be jax-compatible. How can I achieve this?
